packages/core/src/i18n.ts

```diff
@@ -110,9 +124,17 @@ export class I18n extends EventEmitter<Events> {
     }
   }
 
+  /**
+   * @deprecated Plurals automatically used from Intl.PluralRules you can safely remove this call. Introduced in v4
+   */
   public loadLocaleData(allLocaleData: AllLocaleData): void
+  /**
+   * @deprecated Plurals automatically used from Intl.PluralRules you can safely remove this call. Introduced in v4
+   */
```

Comment:
The methods and typings is still here, but has no effect. That's just to make transition a bit easier.